plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/DefaultThriftMetastoreClientFactory.java

```diff
@@ -55,7 +55,6 @@
     private final int timeoutMillis;
     private final HiveMetastoreAuthentication metastoreAuthentication;
     private final String hostname;
-    private final ThriftHiveMetastoreState state;
```

Comment:
in the cmt msg

replace "...from ThriftMetastoreClient."
with "Move alternating calls logic to ThriftHiveMetastore from ThriftMetastoreClient."